tests/fixtures/db.py

```diff
@@ -49,16 +49,16 @@ def project_root() -> Path:
 
 
 @pytest.fixture(scope='session')
-def db_uri(project_root) -> str:
-    with PostgresContainer(POSTGRES_DOCKER_IMAGE) as pg_container:
+def db_port(project_root) -> str:
+    with PostgresContainer(image=POSTGRES_DOCKER_IMAGE) as pg_container:
         database_uri = pg_container.get_connection_url()
 
         config = Config('migrations/alembic.ini')
         with chdir(project_root):
             config.set_main_option('database_uri', database_uri)
             upgrade(config, 'head')
 
-        yield database_uri
+        yield pg_container.get_exposed_port(5432)
```

Comment:
Please change here to default port from constructor:
```suggestion
        yield pg_container.get_exposed_port(pg_container.port_to_expose)
```